network/fabrics_test.go

```diff
@@ -0,0 +1,875 @@
+package network_test
+
+import (
+	"context"
+	"net/http"
+	"path"
+	"strings"
+	"testing"
+
```

Comment:
Whitespace nit: extra newlines.  There's got to be something in GoLand that's not right with respect to its inclusion of `imports`. `goimports` doesn't have this style quirk.